webapp/channels/src/components/widgets/inputs/input/input.tsx

```diff
@@ -151,6 +153,13 @@ const Input = React.forwardRef((
     };
 
     const validateInput = () => {
+        if (validate) {
+            const validationError = validate();
```

Comment:
Could you pass the current value into `validate`? I think that'd simplify the validation functions a bit by making it so that the caller doesn't need to repeat `this.state.myFooBarBaz` in most cases, and it would mean that the validation function could be pure in most cases

Reply:
Yep, that makes sense.